package.json

```
@@ -11,12 +11,11 @@
     "url": "https://github.com/htmlacademy-javascript/1700741-kekstagram-28/issues"
   },
   "devDependencies": {
-    "browser-sync": "2.27.11",
+    "browser-sync": "^2.27.5",
```

Это не нужно было делать
К тому же по критериям все версии должны фиксироваться без модификаторов
```suggestion
    "browser-sync": "2.27.11",
```